test/test_Quaternion.jl

```diff
@@ -29,6 +29,9 @@ let # test rotations
     @test qrotation([0, 0, 0], 1.0) == Quaternion(1.0) # a zero axis should act like zero rotation
     @test qrotation([1, 0, 0], 0.0) == Quaternion(1.0)
     @test qrotation([0, 0, 0]) == Quaternion(1.0)
+    qx = qrotation(view([1, 0, 0], :), view([pi / 4], :)[1])
```

Comment:
I think we don't need `view` in the second argument.
```suggestion
    qx = qrotation(view([1, 0, 0], :), pi / 4)
```